main.py

```diff
@@ -140,24 +142,13 @@ def main():
 
     mqtt_client.subscribe_topics([configuration.mqtt_settings.command_topic])
 
-    def write_to_coil(marshalled_message):
+    def write_to_modbus(message):
```

Comment:
I like removing logic from the main code and put it into dedicated abstractions 👍